lib/models/models.dart

```diff
@@ -30,7 +30,7 @@ class ProductModel extends HiveObject {
       currency: data.data()[PRODUCT_CURRENCY_KEY],
       position: data.data()[PRODUCT_POSITION_KEY] ?? 0,
     );
```

Comment:
Any actual changes on this file or only formatting ? 
It helps when the actual changes are pointed out. Saves the reviewers times.